client/client_eth.go

```diff
@@ -298,3 +298,9 @@ func (c *RpcEthClient) FeeHistory(blockCount rpc.DecimalOrHex, lastBlock types.B
 	err = c.CallContext(context.Background(), &val, "eth_feeHistory", blockCount, lastBlock, rewardPercentiles)
 	return
 }
+
+// TraceTransaction returns the block trace by block number.
```

Comment:
It would be more consistent if we put this into a new file `client_debug.go`

Reply:
yep.